src/ConnectedMode/Migration/IUnintrusiveBindingController.cs

```diff
@@ -0,0 +1,36 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.ComponentModel.Composition;
+
+namespace SonarLint.VisualStudio.ConnectedMode.Migration
+{
+    internal interface IUnintrusiveBindingController
```

Comment:
I spent a good 15 minutes trying to think of a name for this class and eventually gave up, this is a bad name and does not represent what this class does at all however, I am not 100% sure about  1. all the responsibilities & 2. what a sensible name would be for the responsibilities I do know about

I need help thanks

Reply:
I think it's ok to leave it as it is for now, effectively with a placeholder name. When we come to implement it, we'll have a better idea of what its responsibilities are so we can rename it then.

Later on when we come to implement the auto-update of QPs, its responsibilities might change or expand, at which point we might decide to rename it again.